_includes/contact-list.html

```diff
@@ -12,6 +12,15 @@
 	<li>{% include icon-github.html username=site.github_username label='GitHub' %}</li>
 	{% endif %}
 
+	{% if site.wiki %}
+	<li>
+		{% comment %} basically basic theme doesn't have a dokuwiki logo :( {% endcomment %}
+		<a href="{{ site.wiki }}">
+  			<span class="icon"><img src="assets/images/dokuwiki-logo.svg"></span>
```

Comment:
NIT: img is self closing, therefore should have a / at the end:
```suggestion
  			<span class="icon"><img src="assets/images/dokuwiki-logo.svg"/></span>
```

Reply:
> NIT: img is self closing, therefore should have a / at the end:

acshtually, ["Self-closing tags (<tag />) do not exist in HTML."](https://developer.mozilla.org/en-US/docs/Glossary/Void_element#self-closing_tags)

Reply:
> > > NIT: img is self closing, therefore should have a / at the end:
> > 
> > 
> > acshtually, ["Self-closing tags () do not exist in HTML."](https://developer.mozilla.org/en-US/docs/Glossary/Void_element#self-closing_tags)
> 
> NIT: img is a ~self closing~ void element, therefore should have a / at the end. Also, moz themselves use a / at the end of the img tag in their own [examples](https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Elements/img). 😝

NIT: it dun't matter: "[If a trailing / (slash) character is present in the start tag of an HTML element, HTML parsers ignore that slash character.](https://developer.mozilla.org/en-US/docs/Glossary/Void_element#self-closing_tags)"